### ISSUE TITLE
Link for Edit Character Details From Character Log Entry Sheet (Enhancement)

### ISSUE THREAD

**Issue by tjclayton** [Ariel-Thomas/adventurers-league-log#84]
It would be useful (to me at least) to have a link to edit the character details from the character log entry sheet in addition to the option from the character listing grid. For example, if I add enough XP to a character so that the character goes up a level, I have to go back to the character listing, click the button to edit the character details, and then update the "Classes and Levels". An edit link on the character would remove the extra navigation steps.

Definitely low priority.

**Comment by Troacctid** [comment]
This would be useful for me as well.

**Comment by Ariel-Thomas** [comment]
Yeah, this is something I kept thinking of but never implemented. Added to the TODO
